### PR TITLE
NOJIRA: Enable findbugs in Travis CI and Appveyor CI

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] tests are included

##### Description of change

Includes these validation as part of the continuous integration process.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1461,7 +1461,7 @@
                             <plugin>
                                 <groupId>org.codehaus.mojo</groupId>
                                 <artifactId>findbugs-maven-plugin</artifactId>
-                                <version>2.4.0</version>
+                                <version>3.0.5-SNAPSHOT</version>
```
Comment (apetro) [inline pom.xml]: Do we really have to depend on the `SNAPSHOT`? Would prefer to depend on real releases.

Comment (ChristianMurphy) [inline pom.xml]: Sure, I'll lock this to release `3.0.4`

Comment (ChristianMurphy) [inline pom.xml]: patched in https://github.com/Jasig/uPortal/pull/865/commits/17c7a42bef0321267de2d416456553e37207a69b

**Comment by ChristianMurphy** [comment]
Closing this out because majority of the build has moved to gradle which already has findbugs enabled.

**Pull request closed without merging**